src/open_inwoner/openklant/constants.py

```diff
@@ -1,19 +1,18 @@
+from django.db import models
 from django.utils.translation import gettext_lazy as _
 
-from djchoices import ChoiceItem, DjangoChoices
 
-
-class Status(DjangoChoices):
-    nieuw = ChoiceItem("nieuw", _("Nieuw"))
-    in_behandeling = ChoiceItem("in_behandeling", _("In behandeling"))
-    afgehandeld = ChoiceItem("afgehandeld", _("Afgehandeld"))
+class Status(models.TextChoices):
+    nieuw = "nieuw", _("Nieuw")
+    in_behandeling = "in_behandeling", _("In behandeling")
+    afgehandeld = "afgehandeld", _("Afgehandeld")
 
     @classmethod
     def safe_label(cls, value, default=""):
         if not value:
             return default
         try:
-            return cls.labels[value]
+            return dict(cls.choices)[value]
```

Comment:
Same as before, I would prefer this since it's more efficient
```suggestion
            return getattr(cls, value).label
```